.github/workflows/ci.yml

```diff
@@ -66,6 +75,6 @@ jobs:
       - run: until curl -s $SIMPLETEST_BASE_URL; do true; done > /dev/null
 
       - name: Run a single unit test to verify the testing setup.
-        run: ./vendor/bin/phpunit -c ./web/core ./web/core/modules/user/tests/src/Unit/UserAccessControlHandlerTest.php
+        run: ./vendor/bin/phpunit -c ./web/core $(pwd)/web/core/modules/user/tests/src/Unit/UserAccessControlHandlerTest.php
```

Comment:
Latest D11-dev changed the way the relative path discovered. Using absolute for now. 

This change is not relevant to this project's functionality.

https://www.drupal.org/project/drupal/issues/3417066
and
https://www.drupal.org/project/drupal/issues/3445847